test/common.js

```diff
@@ -39,6 +39,7 @@ const DEFAULT_SQRT_PRICE = BigNumber.from(2).pow(96);
 const ONE_WEEK_IN_SECONDS = 7 * 24 * 60 * 60;
 const STANDARD_TOKENS_PER_EUR = 20; // 1 TST = 0.05 EUR
 const DECIMALS = BigNumber.from(10).pow(18);
+const sixDec = BigNumber.from(10).pow(6);
```

Comment:
let's make this all caps since we'll probably see this fella alot in the code from now on 😄.
maybe we should rename the 'standard' 18 decimal const `DECIMALS` too 🤔